### PR TITLE
Lyrics

### PR DESCRIPTION
**Pull request by OJackson1**

### CODE REVIEW

File: Owen.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
+[Verse 1]
```
Comment (Tay-Dzonu-QA) [inline Owen.txt:3]:
```suggestion
[Insert Song Title]

[Verse 1]
```

**Review by Tay-Dzonu-QA — CHANGES_REQUESTED** [review]

**Review by CLC15735 — APPROVED** [review]
Good taste in music my friend

**Review by CodewithOsatQA — APPROVED** [review]
Dope 🤴🏾

**Review by OJackson1 — COMMENTED** [review]
good

**Review by Tay-Dzonu-QA — APPROVED** [review]

**Review by CLC15735 — APPROVED** [review]

**Pull request merged**